examples/cisco-ise-ts/index.ts

```diff
@@ -0,0 +1,111 @@
+import * as ise from "@pulumi/ise";
+
+const username = process.env["ISE_USERNAME"] || "";
+const password = process.env["ISE_PASSWORD"] || "";
+if (username === "" || password === "") {
+    throw new Error("Missing ISE username and/or password");
+}
+
+const url = process.env["ISE_URL"] || "";
+if (url === "") {
+    throw new Error("Missing ISE URL");
+}
+
+const provider = new ise.Provider("example-ise-provider", {
```

Comment:
You don't use this provider anywhere else - is that intended?

Reply:
ahh good catch